host_core/test/host_core_test.exs

```diff
@@ -23,4 +29,29 @@ defmodule HostCoreTest do
     # HOST_ prefix removed.
     assert "42" == labels["testing"]
   end
+
+  test "Host purges actors and providers" do
+    {:ok, bytes} = File.read("priv/actors/echo_s.wasm")
+    {:ok, _pid} = HostCore.Actors.ActorSupervisor.start_actor(bytes)
+
+    {:ok, _pid} =
```

Comment:
This test should also have the `on_exit` calls that are present in other tests just to ensure that the actor and provider are cleaned up at the end

Reply:
I actually left those off because the test itself is testing the ability to purge the host of actors and providers. If the test passes, there are no more providers or actors :)